### PR TITLE
[RFC]  Add rudimentary remote control method 

### PR DESCRIPTION
**Pull request by godsic**
The least intrusive way to control DoA DSP remotely without WEB GUI is to allow clients to read and upload settings.json` file that contains all the DSP settings (not the DAQ settings though). So lets serve `settings.json` on port 8081, where DOA_value.html` is present as well. Since both files have dynamic content, lets remove them from the repo. `settings.json` would be either re-created on start, if not present, or re-used if it is already there. Consequently, `_android_web` folder is also unchecked from the repo. It is re-created on start with a new name, `_share`, to better reflect its purpose.

Technically, the functionality is enabled by switching from PHP to miniserve (https://github.com/svenstaro/miniserve). It not only allows for file uploads, but reduces RAM consumption by roughly 20MB.

The remote control functionallity is hidden behind `en_remote_control` (defaults to 'false') and is subject to `miniserve` avaliability. Disabling remote control also implies server is falling back to PHP for compatibility reasons.

Finally, for the sake of responsiveness the polling rate of the settings file watcher is reduced to 100 ms.

Many thanks to @dubrsl for suggestions.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 ui.log
 .idea/
 __pycache__/
+_android_web/
+**/settings.json*
```
Comment (dubrsl) [inline .gitignore:5]: It will ignore the _nodejs/settings.json too. Not sure that it's correct.

Comment (godsic) [reply .gitignore]: @dubrsl Thanks for pointing out, fixed.

**Comment by godsic** [comment]
Here is an example of how it works in practice:

[Screencast from 2023-07-16 20-39-32.webm](https://github.com/krakenrf/krakensdr_doa/assets/2339291/3b36108f-204e-446e-9209-d08af6dea726)

Of course it could be done from whatever client that can POST a form with `settings.json` content to the Kraken.

**Comment by godsic** [comment]
@krakenrf Please install jq (json parser) with

apt -y install jq

**Comment by krakenrf** [comment]
After installing jq, and setting REMOTE_CONTROL="true", it still appears to revert to php.

Remote Control is DISABLED

And htop shows php running and not miniserve. Any ideas?

**Comment by krakenrf** [comment]
Ah nevermind I see form the code that the core setting is in settings.json

**Comment by godsic** [comment]
@krakenrf It is exclusively controlled by then ‘en_remote_control’ in settings.json to avoid ambiguities.

**Pull request merged**